assets/inbuilt/transformers/readmegenerator/templates/Readme.md

```diff
@@ -4,6 +4,6 @@ Congratulations! Move2Kube has generated the necessary build artfiacts for movin
 
 Next Steps
 ----------
-To try locally use the scripts in the "./scripts" directory, to build, push and deploy. 
+If new dockerfiles were generated by Move2Kube, scripts to build them locally are put in "./scripts" directory. 
 
-For production usage use the CI/CD pipelines for deployment.
\ No newline at end of file
+For production usage use the CI/CD pipelines and artifacts in deploy directory for deployment.
```

Comment:
Can we change "deploy" directory to "./deploy" directory?

Reply:
done

Reply:
Thanks! LGTM